scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/simulation/ResumableSimulationTest.java

```diff
@@ -92,40 +92,6 @@ public void testStopsAtEndOfPlanningHorizon(){
     assert(resumableSimulationDriver.getSimulationResults(Instant.now()).unfinishedActivities.size() == 1);
   }
 
-  @Test
-  public void testThreadsReleased() {
```

Comment:
Why was this test removed? From what I recall, the thread leakiness this was testing wasn't related to Loom

Reply:
As far as I know, there isn't a way to count the number of active virtual threads, i.e. `executor.getActiveCount()`, since virtual threads are multiplexed to OS threads.

Reply:
Another thought, is this test useful now that we are using Loom to manage (virtual) thread lifecycles automatically?

Reply:
Between the advent of Loom, which changes the way that threads work, and the ongoing changes in #1323, this test is no longer particularly meaningful. I've created #1335 to track the gap in testing introduced by removing this test - further discussion of that topic should happen on that ticket.